config/db/mysql-create-tables.sql

```diff
@@ -8,7 +8,7 @@ CREATE TABLE IF NOT EXISTS user (
   tmpblockattempts integer,
   blocked tinyint(1),
   notificationtype varchar(10),
-  notificationaddress varchar(64)
+  notificationaddress varchar(256)
```

Comment:
```suggestion
  notificationaddress varchar(255)
```

If you (arbitrarily) limit the length, probably makes more sense to do it at the 1 byte border instead of 1 over it?